### ISSUE TITLE
Selected features in "AdminDeployment.xml" file don't work

### ISSUE THREAD

**Issue by JulianGindi** [KennethB/Chocolatey-Packages#19]
We are trying to fully automate the installation of Visual Studio 2015 and need to add the Visual C++ additional features (Common Tools for Visual C++ 2015 and Foundation Classes for C++). Even when I select these as "Selected: yes" they don't seem to install. I have attached the AdminDeployment file I am using as well as a screenshot of the non-installed packages. Any advice would be greatly appreciated. 

<img width="459" alt="Screen Shot 2021-03-18 at 1 32 13 PM" src="https://user-images.githubusercontent.com/776248/111693896-6c11bc80-87ee-11eb-9fda-7a46a1e31d7b.png">

Admin Deployment File: https://gist.github.com/JulianGindi/fcf32d8a95e82ef18cacdb6ebcb0c1f4

**Comment by KennethB** [comment]
I haven't used this in a while, but from what I remember you might have to generate a new AdminDeployment file from the current installer.  

https://visualstudio.microsoft.com/vs/support/vs2015/command-line-instructions-visual-studio-2015-installer/

**Comment by JulianGindi** [comment]
Yeah for some reason the installer fails to spit out an AdminDeployment file when I invoke the above command, but that's a me problem, no a choco package issue :) Just thinking the installer might have some bugs I'll need to dig into.

**Comment by JulianGindi** [comment]
A little update. We were able to generate the new AdminDeployment file but it’s still having the same issue. Just wanted to see if you think reaching out to Microsoft is the next step which we are happy to do.

**Comment by KennethB** [comment]
Have you tried manually running the file?

If you look in the package you can see the command it is running
